apps/admin-x-framework/package.json

```diff
@@ -111,7 +111,8 @@
         "dependsOn": [
           "test:unit",
           "^build",
-          "@tryghost/admin-x-design-system:test:unit"
+          "@tryghost/admin-x-design-system:test:unit",
+          "@tryghost/shade:test:unit"
```

Comment:
This means that if the shade tests fail the admin-x-framework tests will fail too, is that what you want?

Reply:
Hm... Maybe not yet. We just added the current design system to it to catch DS bugs, but maybe we can leave Shade out until its first release. Good point.